### PR TITLE
feat: update docs to sdk connectors v2

### PR DESCRIPTION
**Pull request by luizstacio**
On this PR;
- Update documentation and examples for new Connectors API
- Update examples to new Connectors
- Refactor some of the old docs
- Create a new comment `// ignore-line` to ignore lines on the the code examples show on the docs.

closes: FE-50

### CODE REVIEW

File: packages/docs/docs/dev/signMessage.mdx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+<CodeImport
+  file="../../examples/signMessage/SignMessage.tsx"
+  commentBlock="signMessage"
+/>
+<Examples.SignMessage />
+
+## With React
+
+In a react app, once the connection is established, you can use the `useWallet` hook to get a wallet instance and sign the transaction.
```
Comment (calldelegation) [inline packages/docs/docs/dev/signMessage.mdx:18]:
```suggestion
In a react app, once the connection is established, you can use the `useWallet()` hook to get a wallet instance and sign the transaction.
```

File: packages/docs/docs/dev/reference.mdx [hunk 2]
```diff
@@ -194,55 +155,20 @@ The events enum `FuelWalletEvents` can be imported from the `@fuel-wallet/sdk` p
...
+
+| Event              | Trigger                   | Params                 |
+| ------------------ | ------------------------- | ---------------------- |
+| `connectors`       | connectors change         | `Array<FuelConnector>` |
+| `currentConnector` | currentConnector change   | `FuelConnector`        |
+| `accounts`         | accounts available change | `Array<string>`        |
+| `currentAccount`   | current account change    | `string \| null`       |
+| `connection`       | connection state change   | `boolean`              |
+| `currentNetwork`   | cuurent network change    | `Network`              |
+| `assets`           | assets metadata change    | `Array<Asset>`         |
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:174]: Chart looks very odd when generated
Screenshot 2023-11-22 at 5 02 57 PM
Should match the docs-hub charts
Screenshot 2023-11-22 at 5 06 28 PM

File: packages/docs/docs/dev/reference.mdx [hunk 3]
```diff
@@ -7,180 +7,141 @@ category: For Developers
 
 ## Methods
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:8]:
```suggestion
```

Should remove and bump everything down a tag to match the hooks reference which looks a lot cleaner

Screenshot 2023-11-22 at 5 15 15 PM

File: packages/docs/docs/dev/reference.mdx [hunk 4]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
 
+### Listing connectors
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:10]:
```suggestion
## Listing connectors
```

File: packages/docs/docs/dev/reference.mdx [hunk 5]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
+
+`connectors(): Promise<Array<FuelConnector>>`<br />
+Return connectors with availability metadata.
+
+### Has connector
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:15]:
```suggestion
## Has connector
```

File: packages/docs/docs/dev/reference.mdx [hunk 6]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
+
+`hasConnector(connectorName?: string | FuelConnector): Promise<boolean>`<br />
+Check if any Connector is avialable or if a connector is provided check if it is available.
+
+### Current connector
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:20]:
```suggestion
## Current connector
```

File: packages/docs/docs/dev/reference.mdx [hunk 7]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
+
+`currentConnector(): FuelConnector`<br />
+Return the current selected connector.
+
+### Get connector
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:25]:
```suggestion
## Get connector
```

File: packages/docs/docs/dev/reference.mdx [hunk 8]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
+
+`getConnector(connectorName: string): FuelConnector`<br />
+Return the connector with the given name.
+
+### Selecting connector
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:30]:
```suggestion
## Selecting connector
```

File: packages/docs/docs/dev/reference.mdx [hunk 9]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
+
+`selectConnector(connectorName: string): Promise<boolean>` <br />
+This method will check if the desired connector is installed. If it is not detected in 1 second, the method throws an error.
+
 ### Version
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:35]:
```suggestion
## Version
```

File: packages/docs/docs/dev/reference.mdx [hunk 10]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
 
-`async version(): Promise<string>`<br />
-Return the current version of the Wallet.
-testCase="version" />
+`version(): Promise<string>`<br />
+Return the current version of the Wallet and Network supported.
 
 ### Connect
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:40]:
```suggestion
## Connect
```

File: packages/docs/docs/dev/reference.mdx [hunk 11]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
 
-`async connect(): Promise<boolean>`<br />
-Request permission to start a connection between the project and the wallet
-
-{/* <CodeImport file="../../../sdk/src/tests/Fuel.test.ts" testCase="connect" /> */}
+`connect(): Promise<boolean>`<br />
+Request permission to start a connection between the project and the wallet. Return the connection state.
 
 ### Disconnect
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:45]:
```suggestion
## Disconnect
```

File: packages/docs/docs/dev/reference.mdx [hunk 12]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
 
-`async disconnect(): Promise<boolean>`<br />
-Disconnect your project
-
-{/* <CodeImport file="../../../sdk/src/tests/Fuel.test.ts" testCase="disconnect" /> */}
+`disconnect(): Promise<boolean>`<br />
+Disconnect your project from the wallet. Return the connection state.
 
 ### Get connection status
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:50]:
```suggestion
## Get connection status
```

File: packages/docs/docs/dev/reference.mdx [hunk 13]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
 
-`async isConnected(): Promise<boolean>`<br />
+`isConnected(): Promise<boolean>`<br />
 Return the state of the application connection.
 
-{/* <CodeImport file="../../../sdk/src/tests/Fuel.test.ts" testCase="isConnected" /> */}
-
 ### List Accounts
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:55]:
```suggestion
## List accounts
```

File: packages/docs/docs/dev/reference.mdx [hunk 14]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
 
-`async accounts(): Promise<Array<string>>`<br />
-List accounts in the wallet
-
-{/* <CodeImport file="../../../sdk/src/tests/Fuel.test.ts" testCase="accounts" /> */}
+`accounts(): Promise<Array<string>>`<br />
+Return the accounts authorized for the current connection.
 
 ### Current Account
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:60]:
```suggestion
## Current account
```

File: packages/docs/docs/dev/reference.mdx [hunk 15]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
 
-`async currentAccount(): Promise<string>`<br />
+`currentAccount(): Promise<string | null>`<br />
 Return the current account being used in the wallet application.
-
-{/* <CodeImport file="../../../sdk/src/tests/Fuel.test.ts" testCase="currentAccount" /> */}
+If the account selected is not authorized for the connection, returns null.
 
 ### List Networks
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:66]:
```suggestion
## List networks
```

File: packages/docs/docs/dev/reference.mdx (hunk 3 above)
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:8]: Can we get the methods here in alphabetical order please?

File: packages/docs/docs/dev/reference.mdx [hunk 16]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
 
-`async networks(): Promise<Array<Network>>`<br />
+`networks(): Promise<Array<Network>>`<br />
 Return the list of networks in the current wallet.
 
-{/* <CodeImport file="../../../sdk/src/tests/Fuel.test.ts" testCase="networks" /> */}
-
 ### Current Network
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:71]:
```suggestion
## Current network
```

File: packages/docs/docs/dev/reference.mdx [hunk 17]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
 
-`async network(): Promise<Network>`<br />
+`currentNetwork(): Promise<Network>`<br />
 Return the current network being used in the wallet application.
 
-{/* <CodeImport file="../../../sdk/src/tests/Fuel.test.ts" testCase="network" /> */}
+### Select Network
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:76]:
```suggestion
## Select network
```

File: packages/docs/docs/dev/reference.mdx [hunk 18]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
 
-### Add Network
+`selectNetwork(url: string): Promise<boolean>`<br />
+Request the user to switch to a different network. Return true if success or false it fails.
 
-`async addNetwork(network: Network): Promise<boolean>`<br />
-Add network to the wallet.
+### Add Network
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:81]:
```suggestion
## Add network
```

File: packages/docs/docs/dev/reference.mdx [hunk 19]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
 
-{/* <CodeImport file="../../../sdk/src/tests/Fuel.test.ts" testCase="addNetwork" /> */}
+`addNetwork(url: string): Promise<boolean>`<br />
+Request the user to add a new network. Return true if success or false it fails.
 
 ### List Assets
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:86]:
```suggestion
## List assets
```

File: packages/docs/docs/dev/reference.mdx [hunk 20]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
 
-`async assets(): Promise<Array<Asset>>`<br />
+`assets(): Promise<Array<Asset>>`<br />
 Return the list of assets in the current wallet.
 
-{/* <CodeImport file="../../../sdk/src/tests/Fuel.test.ts" testCase="assets" /> */}
-
 ### Add Asset
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:91]:
```suggestion
## Add asset
```

File: packages/docs/docs/dev/reference.mdx [hunk 21]
```diff
@@ -194,55 +155,20 @@ The events enum `FuelWalletEvents` can be imported from the `@fuel-wallet/sdk` p
...
-  file="../../examples/Connectors.tsx"
-  commentBlock="eventConnectors"
-/>
+```ts
+fuel.on(fuel.events.connection, (connectionState) => {
+  console.log(connectionState);
+});
+```
+
+### Events
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:164]: Is there a better name for this? "Events" under "Events" doesnt really make sense

File: packages/docs/docs/dev/reference.mdx [hunk 22]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
-{/* <CodeImport file="../../../sdk/src/tests/FuelWalletConnectors.test.ts" testCase="addConnector" /> */}
+### Destory
 
-### Remove connector
-
-To remove a Wallet Connector use the `removeConnector(connectorName: string): boolean` method.
-
-{/* <CodeImport file="../../../sdk/src/tests/FuelWalletConnectors.test.ts" testCase="removeConnector" /> */}
+`destroy(): void`<br />
+Removes all listeners and clean the storage.
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:144]:
```suggestion
Removes all listeners and cleans the storage.
```

File: packages/docs/docs/dev/reference.mdx [hunk 23]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
-
-{/* <CodeImport file="../../../sdk/src/tests/FuelWalletConnectors.test.ts" testCase="selectConnector" /> */}
-
-### Add connector
-
-For adding a Wallet Connector use the `addConnector(connector: FuelWalletConnector): boolean` method.
-This method does not trigger any errors if the target Connector is not installed.
-
-{/* <CodeImport file="../../../sdk/src/tests/FuelWalletConnectors.test.ts" testCase="addConnector" /> */}
+### Destory
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:141]:
```suggestion
## Destory
```

File: packages/docs/docs/dev/reference.mdx [hunk 24]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
-
-### Has connector
-
-To check if a connector is on the connectors list use the `hasConnector(connectorName: string): boolean` method.
-
-{/* <CodeImport file="../../../sdk/src/tests/FuelWalletConnectors.test.ts" testCase="hasConnector" /> */}
-
-### Selecting connector
+`clean(): void`<br />
+Clean the storage from Fuel SDK. Used to keep track of selected connector.
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:139]:
```suggestion
Clean the storage used by the Fuel SDK, which is utilized to keep track of the selected connector.
```

File: packages/docs/docs/dev/reference.mdx [hunk 25]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
 
-{/* <CodeImport file="../../../sdk/src/tests/Fuel.test.ts" testCase="getProvider" /> */}
+### Unsubscribe all
 
-#### Using provider on a fuels-ts Wallet
+`unsubscribe(): void`<br />
+Remove all open listeners this is useful when you want to remove the Fuel instance and avoid memory leaks.
 
-{/* <CodeImport file="../../../sdk/src/tests/Fuel.test.ts" testCase="User getProvider on fuels-ts Wallet" /> */}
+### Destory
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:136]:
```suggestion
## Clean
```

Wrong title

File: packages/docs/docs/dev/reference.mdx [hunk 26]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
-#### Using provider to query node info
+`getWallet(address: string | AbstractAddress, provider?: FuelWalletProvider): Promise<FuelWalletLocked>`<br />
+Return a `FuelWalletLocked` instance, witch can be used for contract, transfers and other interactions.
 
-{/* <CodeImport file="../../../sdk/src/tests/Fuel.test.ts" testCase="getProvider" /> */}
+### Unsubscribe all
 
-#### Using provider on a fuels-ts Wallet
+`unsubscribe(): void`<br />
+Remove all open listeners this is useful when you want to remove the Fuel instance and avoid memory leaks.
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:134]:
```suggestion
Remove all open listeners. This is useful when you want to dispose of the Fuel instance and avoid memory leaks.
```

File: packages/docs/docs/dev/reference.mdx [hunk 27]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
-
-Note: The provider returned is tied to the current network selected by the user in their wallet. This means that if the user changes the network within their wallet,
-the DApp will also switch to that network. Support for specific DApp networks will be available in the future.
-
-#### Using provider to query node info
+`getWallet(address: string | AbstractAddress, provider?: FuelWalletProvider): Promise<FuelWalletLocked>`<br />
+Return a `FuelWalletLocked` instance, witch can be used for contract, transfers and other interactions.
 
-{/* <CodeImport file="../../../sdk/src/tests/Fuel.test.ts" testCase="getProvider" /> */}
+### Unsubscribe all
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:131]:
```suggestion
## Unsubscribe all
```

File: packages/docs/docs/dev/reference.mdx [hunk 28]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
-`getProvider(): Promise<FuelWalletProvider>`<br />
-Return a FuelWalletProvider. This class extends the fuels-ts SDK's `Provider`,
-enabling all of the methods available for `Provider` while using the FuelSDK on signature points to request user permissions.
-
-Note: The provider returned is tied to the current network selected by the user in their wallet. This means that if the user changes the network within their wallet,
-the DApp will also switch to that network. Support for specific DApp networks will be available in the future.
-
-#### Using provider to query node info
+`getWallet(address: string | AbstractAddress, provider?: FuelWalletProvider): Promise<FuelWalletLocked>`<br />
+Return a `FuelWalletLocked` instance, witch can be used for contract, transfers and other interactions.
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:129]:
```suggestion
Return a `FuelWalletLocked` instance, which can be used for contracts, transfers, and other interactions.
```

File: packages/docs/docs/dev/reference.mdx [hunk 29]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
-
 ### Send transaction
 
-`async sendTransaction(transaction: TransactionRequestLike): Promise<string>`<br />
-Send a transaction, this will request the user selected account to review,
-sign, and send the transaction.
-
-{/* <CodeImport file="../../../sdk/src/tests/Fuel.test.ts" testCase="sendTransaction" /> */}
+`async sendTransaction(address: string, transaction: TransactionRequestLike): Promise<string>`<br />
+Request the user specific account to send a transaction.
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:124]:
```suggestion
Request a specific user account to send a transaction.
```

File: packages/docs/docs/dev/reference.mdx [hunk 30]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
 
 ### Get Wallet
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:126]:
```suggestion
## Get Wallet
```

File: packages/docs/docs/dev/reference.mdx [hunk 31]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
+Add ABI to the users Wallet.
 
 ### Request signature message
 
 `async signMessage(address: string, message: string): Promise<string>`<br />
 Request a message signature for one specific account
 
-{/* <CodeImport file="../../../sdk/src/tests/Fuel.test.ts" testCase="signMessage" /> */}
-
 ### Send transaction
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:121]:
```suggestion
## Send transaction
```

File: packages/docs/docs/dev/reference.mdx [hunk 32]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
 ### Add ABI
 
-`async addAbi(abiMap: AbiMap): Promise<boolean>`<br />
-Add ABI to the wallet.
-
-{/* <CodeImport file="../../../sdk/src/tests/Fuel.test.ts" testCase="addAbi" /> */}
+`addAbi(abiMap: AbiMap): Promise<boolean>`<br />
+Add ABI to the users Wallet.
 
 ### Request signature message
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:116]:
```suggestion
## Request signature message
```

File: packages/docs/docs/dev/reference.mdx [hunk 33]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
 ### Has ABI
 
-`async hasAbi(contractId: string): Promise<boolean>`<br />
-Check if has ABI for contractId given.
-
-{/* <CodeImport file="../../../sdk/src/tests/Fuel.test.ts" testCase="hasAbi" /> */}
+`hasABI(contractId: string): Promise<boolean>`<br />
+Return boolean if the ABI for contractId given is found.
 
 ### Add ABI
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:111]:
```suggestion
## Add ABI
```

File: packages/docs/docs/dev/reference.mdx [hunk 34]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
 ### Get ABI
 
-`async getAbi(contractId: string): Promise<JsonAbi>`<br />
-Get ABI of contractId given.
-
-{/* <CodeImport file="../../../sdk/src/tests/Fuel.test.ts" testCase="getAbi" /> */}
+`getABI(contractId: string): Promise<JsonAbi>`<br />
+Return ABI of contractId given or null if not found.
 
 ### Has ABI
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:106]:
```suggestion
## Has ABI
```

File: packages/docs/docs/dev/reference.mdx [hunk 35]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
 ### Add Asset
 
-`async addAsset(asset: Asset): Promise<boolean>`<br />
-Add asset to the wallet.
-
-{/* <CodeImport file="../../../sdk/src/tests/Fuel.test.ts" testCase="addAsset" /> */}
+`addAsset(asset: Asset): Promise<boolean>`<br />
+Request the user to add a asset metadata to the wallet.
 
 ### Add Assets
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:96]:
```suggestion
## Add assets
```

File: packages/docs/docs/dev/reference.mdx [hunk 36]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
 
-`async addAssets(asset: Asset[]): Promise<boolean>`<br />
-Add assets to the wallet.
-
-{/* <CodeImport file="../../../sdk/src/tests/Fuel.test.ts" testCase="addAssets" /> */}
+`addAssets(asset: Asset[]): Promise<boolean>`<br />
+Request the user to add multiple assets metadata at once to the wallet.
 
 ### Get ABI
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:101]:
```suggestion
## Get ABI
```

File: packages/docs/docs/dev/reference.mdx [hunk 37]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
+Return boolean if the ABI for contractId given is found.
 
 ### Add ABI
 
-`async addAbi(abiMap: AbiMap): Promise<boolean>`<br />
-Add ABI to the wallet.
-
-{/* <CodeImport file="../../../sdk/src/tests/Fuel.test.ts" testCase="addAbi" /> */}
+`addAbi(abiMap: AbiMap): Promise<boolean>`<br />
+Add ABI to the users Wallet.
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:114]:
```suggestion
Add the ABI to the user's wallet.
```

File: packages/docs/docs/dev/reference.mdx [hunk 38]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
+Return ABI of contractId given or null if not found.
 
 ### Has ABI
 
-`async hasAbi(contractId: string): Promise<boolean>`<br />
-Check if has ABI for contractId given.
-
-{/* <CodeImport file="../../../sdk/src/tests/Fuel.test.ts" testCase="hasAbi" /> */}
+`hasABI(contractId: string): Promise<boolean>`<br />
+Return boolean if the ABI for contractId given is found.
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:109]:
```suggestion
Return a boolean indicating if the ABI for the given contractId is found.
```

File: packages/docs/docs/dev/reference.mdx [hunk 39]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
+Request the user to add multiple assets metadata at once to the wallet.
 
 ### Get ABI
 
-`async getAbi(contractId: string): Promise<JsonAbi>`<br />
-Get ABI of contractId given.
-
-{/* <CodeImport file="../../../sdk/src/tests/Fuel.test.ts" testCase="getAbi" /> */}
+`getABI(contractId: string): Promise<JsonAbi>`<br />
+Return ABI of contractId given or null if not found.
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:104]:
```suggestion
Return the ABI of the given contractId, or null if not found.
```

File: packages/docs/docs/dev/reference.mdx [hunk 40]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
+Request the user to add a asset metadata to the wallet.
 
 ### Add Assets
 
-`async addAssets(asset: Asset[]): Promise<boolean>`<br />
-Add assets to the wallet.
-
-{/* <CodeImport file="../../../sdk/src/tests/Fuel.test.ts" testCase="addAssets" /> */}
+`addAssets(asset: Asset[]): Promise<boolean>`<br />
+Request the user to add multiple assets metadata at once to the wallet.
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:99]:
```suggestion
Request the user to add metadata for multiple assets to the wallet at once.
```

File: packages/docs/docs/dev/reference.mdx [hunk 41]
```diff
@@ -7,180 +7,141 @@ category: For Developers
...
-{/* <CodeImport file="../../../sdk/src/tests/Fuel.test.ts" testCase="assets" /> */}
-
 ### Add Asset
 
-`async addAsset(asset: Asset): Promise<boolean>`<br />
-Add asset to the wallet.
-
-{/* <CodeImport file="../../../sdk/src/tests/Fuel.test.ts" testCase="addAsset" /> */}
+`addAsset(asset: Asset): Promise<boolean>`<br />
+Request the user to add a asset metadata to the wallet.
```
Comment (calldelegation) [inline packages/docs/docs/dev/reference.mdx:94]:
```suggestion
Request the user to add metadata for an asset to the wallet.
```

**Review by calldelegation — CHANGES_REQUESTED** [review]
We have already discussed the need for a spellcheck linter with @luizstacio, but it won't be included in this PR.

Regarding testing some of the code blocks:
1. Are we waiting for the beta-5 links?
2. Where is the new wallet that will be compatible with these code blocks?
Screenshot 2023-11-22 at 5 43 52 PM

File: packages/docs/docs/dev/abis.mdx [hunk 42]
```diff
@@ -7,16 +7,16 @@ category: For Developers
...
 
-To get the ABI of a contract ID use the `getAbi('contractId')` method on the `window.fuel` object.
+To get the ABI of a contract ID use the `getAbi` with `contractId`.
 
-<CodeImport file="../../examples/Abi.tsx" />
+<CodeImport file="../../examples/abis/Abi.tsx" commentBlock="getABI" />
 
 <Examples.Abi />
 
 ## Adding ABI
```
Comment (calldelegation) [inline packages/docs/docs/dev/abis.mdx:16]: Makes more sense to put "Adding ABI" first and "Get ABI of contract ID" after. I was clicking get ABI of contracts and it returns null and displays nothing (as expected) but developers new to blockchain wouldnt know that.

File: packages/docs/docs/dev/abis.mdx [hunk 43]
```diff
@@ -7,16 +7,16 @@ category: For Developers
 
 ## Get ABI of contract ID
 
-To get the ABI of a contract ID use the `getAbi('contractId')` method on the `window.fuel` object.
+To get the ABI of a contract ID use the `getAbi` with `contractId`.
```
[the remark this answers is not in the document]
Comment (calldelegation) [reply packages/docs/docs/dev/abis.mdx:10]: Can you add a note that you cannot derive an ABI from just a contract ID? New developers might not know this.

File: packages/docs/docs/dev/connectors.mdx [hunk 44]
```diff
@@ -3,78 +3,75 @@ title: Wallet Connectors
...
 
-If you are using our Communication SDK [Content Page Script](https://developer.chrome.com/docs/extensions/mv3/content_scripts/) import `ContentProxyConnection` and pass the name of Wallet Extension.
+You can see the full a DApp example on the [examples](https://github.com/FuelLabs/fuels-wallet/tree/master/examples) folder.
 
-> Otherwise, remember to filter events by the ones that have your Wallet Connector Name as the target.
+### Only hooks
 
-```ts
-import { ContentProxyConnection } from '@fuel-wallet/sdk';
+If you prefer to build your own UI for the connectors you can achive the experience using
```
Comment (calldelegation) [inline packages/docs/docs/dev/connectors.mdx:69]: Screenshot 2023-11-23 at 10 47 07 AM
Fuelet connection opens up the Fuel Wallet instead

File: packages/docs/docs/dev/connectors.mdx [hunk 45]
```diff
@@ -3,78 +3,75 @@ title: Wallet Connectors
...
-### CurrentConnector
-
-This event is triggered when the Current Connector is updated. The callback function receives the `FuelWalletConnector` object.
+Once you have a list of connectors, you can allow the user to select the connector he want to use. By using the methoid `selectConnect`.
+If the connector is not installed, the SDK will return false.
 
 <CodeImport
-  file="../../examples/Connectors.tsx"
-  commentBlock="eventCurrentConnector"
+  file="../../examples/connectors/SelectConnector.tsx"
```
Comment (calldelegation) [inline packages/docs/docs/dev/connectors.mdx:42]: Screenshot 2023-11-23 at 10 46 06 AM
Not detecting Fuelet wallet or the non-development version of Fuel wallet

File: packages/docs/docs/dev/connectors.mdx [hunk 46]
```diff
@@ -3,78 +3,75 @@ title: Wallet Connectors
 category: For Developers
 ---
 
-# Wallet Connectors
+# Fuel Wallet Connectors
 
-A Wallet Connector is an interface to integrate with third-party wallets on the Fuel Ecosystem. It exposes all currently installed wallets in a unified manner
-via `window.fuel`.
+Fuel Wallet Connectors are a interface provided by Wallet developers to allow, you DApp to integrate with specific Wallets.
```
[the remark this answers is not in the document]
Comment (luizstacio) [reply packages/docs/docs/dev/connectors.mdx:8]: Fuel Wallet, Fuellet or anyother one that can be develop in the future/

File: packages/docs/docs/dev/connectors.mdx (hunk 45 above)
Comment (luizstacio) [reply packages/docs/docs/dev/connectors.mdx:42]: They will need to update the wallet to work with the new connectors, we require a new method ping and version.

File: packages/docs/docs/dev/reference.mdx (hunk 21 above)
Comment (calldelegation) [reply packages/docs/docs/dev/reference.mdx:164]: Used "Event Types" for the chart

**Review by calldelegation — APPROVED** [review]

**Review by calldelegation — APPROVED** [review]
LFG can't wait for this :)

**Pull request merged**